src/src/pages/capabilities/NewCapabilityDialog.js

```diff
@@ -39,6 +69,18 @@ export default function NewCapabilityDialog({
     setFormData((prev) => ({ ...prev, ...{ description: newValue } }));
   };
 
+  async function changeInvitation(e) {
+    e.preventDefault();
+    const regex = /[^,]*$/;
```

Comment:
maybe a comment what this regex does?

Reply:
Addressed in https://github.com/dfds/selfservice-portal/pull/133/commits/cb4b6bfc44c8d7dff3c4ea8e1c66e2dc416c35bd